bindings/python/pymongocrypt/binding.py

```diff
@@ -57,11 +57,8 @@ def _parse_version(version):
  * See all public API documentation in: @ref mongocrypt.h
  */
 
-/**
- * @def MONGOCRYPT_VERSION
- * The version string describing libmongocrypt.
- * Has the form x.y.z-<pre>+<date>+git<sha>.
- */
+/* clang-format off */
+/* clang-format on */
```

Comment:
What's the point of turning `clang-format` off and on in succession?

Reply:
This string is generated by a script that strips certain content from `mongocrypt.h`. In this case, the lines removed were all C preprocessor directives, but these two surrounding comments remain.